test/service/test_app_logging.py

```diff
@@ -35,13 +32,12 @@ def test_request_logs(self):
         for level in INFO, DEBUG:
             for authenticated in False, True:
                 with self.subTest(level=level, authenticated=authenticated):
-                    url = self.base_url + '/health/basic'
                     headers = {'authorization': 'Bearer foo_token'} if authenticated else {}
                     with self.assertLogs(logger=log, level=level) as logs:
-                        requests.get(url, headers=headers)
+                        requests.get(self.base_url.set(path='/health/basic').url, headers=headers)
                     logs = [(r.levelno, r.getMessage()) for r in logs.records]
                     headers = {
-                        'host': furl(self.base_url).netloc,
+                        'host': self.base_url.netloc,
```

Comment:
Undo the inlining of the the `url` variable and then 
```suggestion
                        'host': url.netloc,
```